### DISCUSSION TITLE
controller의 ip를 domain으로 설정할 수 없을까요?

### DISCUSSION THREAD

**Discussion by youtomeTiger** [naver/ngrinder#750] [Q&A] [answered]
https://github.com/naver/ngrinder/blob/develop/ngrinder-core/src/main/java/org/ngrinder/NGrinderAgentStarter.java#L135 를 보면 domain으로 넣어도 처음 로딩할때 ip로 변경해서 동적 domain으로 넣은 의미가 없어집니다.
AWS와 같은 환경에서 앞에 LB를 두고 구성하면 연결할 수 없겠더군요.

log를 보면 connection이 끊어지면 1분마다 다시 접속하도록 되어 있는데요. 이때 domain을 다시 lookup 하도록 하는게 좋겠습니다.

의견 부탁드립니다.

**Comment by junoyoon** [comment, marked as answer]
Thx. Your request is quite reasonable.

@donggyu04 plz check it.

**Reply by donggyu04** [reply]
@junoyoon @youtomeTiger 

Thanks for your feedback.
I will check & fix it for use in the next release :)

**Reply by youtomeTiger** [reply]
Thank you so much.

**Reply by donggyu04** [reply]
@youtomeTiger 

Relevant [feature](https://github.com/naver/ngrinder/pull/756) has been released.

**Reply by youtomeTiger** [reply]
thank you.
